infos/src/main.rs

```diff
@@ -1138,6 +1138,7 @@ fn word_pair_proximities_docids(
     prefix.extend_from_slice(word1.as_bytes());
     prefix.push(0);
     prefix.extend_from_slice(word2.as_bytes());
+    prefix.push(0);
```

Comment:
Are we sure that we want to add this byte at the end, this way we change the serialization format and deserializers must be all synchronized.

Reply:
I talked a little bit with Many about it, and I think the answer is that it is not strictly needed. I added it there initially so that we can be sure that the keys are lexicographically ordered tuples (word1, word2, proximity).

But the proximity is always very small (= ~3 I think?), and the tokeniser removes all character bytes from the word tokens, so that they cannot contains bytes smaller than 3. So, almost by accident, the lexicographical order is respected anyway.

I would still rather make it explicit though. If we ever accept larger proximities, or character bytes of `0x1, 0x2, 0x3`, then it will cause very confusing bugs.